### PR TITLE
Documentation - add named code snippet tags

### PR DESCRIPTION
**Pull request by diberry**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: quickstarts/c#/PersonalizerExample/Program.cs [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
 
 namespace PersonalizerExample
 {
     class Program
     {
-        // The key specific to your personalizer service instance; e.g. "0123456789abcdef0123456789ABCDEF"
-        private const string ApiKey = "";
+        // <classVariables>
+        // The key specific to your personalizer resource instance; e.g. "0123456789abcdef0123456789ABCDEF"
+        private const string ApiKey = "9e0684f792f94b858d211e49de169474";//Environment.GetEnvironmentVariable("PERSONALIZER_RESOURCE_KEY");
```
Comment (tyclintw) [inline quickstarts/c#/PersonalizerExample/Program.cs]: Looks like you accidentally submitted your key. I suggest removing this and rotating your keys.

Comment (diberry) [reply quickstarts/c#/PersonalizerExample/Program.cs]: Thank you for letting me know. Removed those.

**Comment by diberry** [comment]
@tyclintw Is this one of those changes that a squash merge is not going to show the key or will is show the key then turn into a requirement to scrub the commit?

**Comment by tyclintw** [comment]
> @tyclintw Is this one of those changes that a squash merge is not going to show the key or will is show the key then turn into a requirement to scrub the commit?

Pull Requests live forever, so you'll need to have it scrubbed. You can always rotate the key so it doesn't matter that it's published.

**Comment by diberry** [comment]
@tyclintw The only way I know to scrub the pii is to use git filter-branch, which removes the file from all history. Then add the file back. Is this how you would do it?

**Comment by tyclintw** [comment]
As far as I know it's pretty much impossible to scrub a pull request without contacting GitHub. See the "Warning" section [here](https://help.github.com/en/articles/removing-sensitive-data-from-a-repository)

**Pull request closed without merging**